app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -3039,7 +3039,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
         menu.findItem(R.id.showreminders).setVisible(Pref.getBoolean("plus_show_reminders", true) && !is_newbie);
 
         LibreBlock libreBlock = null;
-        if (DexCollectionType.hasLibre()) {
+        if (DexCollectionType.hasLibre() || get_follower()) {
```

Comment:
I think this additional use of get_follower needs to also be conditional on a preference switch to avoid this being activated for all followers. eg non-libre etc

Reply:
Is it OK if I'll have this checked once per hour (with a rate limiter) or do you still prefer that I'll add a switch?